### DISCUSSION TITLE
Creating templates usable by mailgun

### DISCUSSION THREAD

**Discussion by magick93** [maizzle/framework#830] [Q&A] [answered]
Hello

I'm trying to produce email templates that can be populated using json, using a handlebars like templating engine.

In my template, I'm using in the frontmatter, for example:

```
---
title: "{{{ first_name }}} Confirm your email address"
preheader: "Please confirm your email address in order to activate your account"
bodyClass: bg-slate-50
---
```

But the production build is producing:
```
<title>undefined} Confirm your email address</title>
```

Similarly, if I use `title: "@{{first_name}} Confirm your email address"` in the frontmatter, the production build shows both ` <title>undefined Confirm your email address</title>` and, correctly, the following:
 ```
<p style="margin: 0; margin-bottom: 24px">
{{first_name}} Confirm your email address by clicking the button below:
 </p>
```

If  I use `title: "@@{{first_name}} ` in the frontmatter, the `title` works as I'd hope - `<title>{{first_name}} Confirm your email address</title>` but other usage of the `title` frontmatter var is:
```
<p style="margin: 0; margin-bottom: 24px">
                        @{{first_name}} Confirm your email address by clicking the button below:
                      </p>
```

How can I have the final production output contain handlebar placeholders?

Thanks

**Comment by cossssmin** [comment, marked as answer]
Try using `@{{ }}` in front matter and layouts. If you have `@{{ }}` in front matter, `{{ }}` should work in a template.
